source/math/IntRandomGenerator.ooc

```diff
@@ -15,17 +15,35 @@
 // along with this program. If not, see <http://www.gnu.org/licenses/>.
 
 use ooc-math
+import math
 import os/Time
 
-IntRandomGenerator: class {
-	_backend: FloatRandomGenerator
-	init: func ~withBackend (=_backend)
-	free: override func {
-		this _backend free()
-		super()
+IntUniformRandomGenerator: class {
+	_state, _min, _max, _range: Int
+	init: func (seed := Time microtime()) {
+		this _min = 0
+		this _max = 32767
```

Comment:
Ah, I see now, you're right. We could have the default value of _max be INT_MAX.
Is it possible (easy) to extend Short in a nice way so that next~fast() returns Short but works well (casts automatically) with Int variables?
Otherwise I'll make next() the only function, and it will use the faster version if range < 32767, to avoid all confusion.